main_test.go

```diff
@@ -51,15 +51,15 @@ func TestBinarySize(t *testing.T) {
 	// so.
 	//
 	// When increasing, use current binary size on amd64 + 1M.
-	const maxSize int64 = 15000000
+	const maxSize int64 = 9103464
 
 	cmd := exec.Command("go", "version")
 	version, err := cmd.CombinedOutput()
 	require.NoError(t, err)
 	t.Logf("Go version: %s", string(version))
 
 	programName := "/tmp/mender"
-	cmd = exec.Command("go", "build", "-o", programName)
+	cmd = exec.Command("go", "build", "-tags", "cli_minimal", "-o", programName)
```

Comment:
This can be removed now, can't it?

Reply:
Thanks, good catch. I missed cleaning this one in the rework.